MetaCell/salk-interactive-atlas
# Feature/224

Closes #224 

Implemented solution: 
- Adds about modal

How to test this PR: 
Click on learn more in the sidebar and check if the modal shows up as desired:
![image](https://user-images.githubusercontent.com/19196034/200633929-7d619b3a-dbea-41d4-83d8-1895e2526db6.png)


### Sanity checks:

### Breaking changes (select one):

### Possible deployment updates issues (select one):

### Test coverage (select one):

### Documentation (select one):
